.github/workflows/build-and-deploy.yml

```diff
@@ -191,7 +151,7 @@ jobs:
       name: production
       url: ${{ steps.deploy.outputs.environment_url }}
     concurrency: deploy_production
-    needs: [docker, deploy_nonprod]
+    needs: [docker]
```

Comment:
should be needs: [docker, deploy_v2_nonprod]
and also remove "continue-on-error: true" from deploy_v2_nonprod

Reply:
Fixed